From bd8ca7573c3b840846f346a97de927fec2d04d6b Mon Sep 17 00:00:00 2001
From: Daniel-Constantin Mierla <miconda@gmail.com>
Date: Wed, 25 Jan 2017 14:45:51 +0100
Subject: [PATCH] htable: use custom column name for order by in db load

- reported by Sebastian Damm, GH #948

(cherry picked from commit a9ffc2a1f7514a7bd4ff0e294e3bcb50d3a2bf07)
---
 modules/htable/ht_db.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/htable/ht_db.c b/modules/htable/ht_db.c
index e1ab7daa7..d5a5a548f 100644
--- a/modules/htable/ht_db.c
+++ b/modules/htable/ht_db.c
@@ -204,6 +204,7 @@ int ht_db_load_table(ht_t *ht, str *dbtable, int mode)
 		return -1;
 	}
 	if(ht->ncols>0) {
+		db_ord = &ht->scols[0];
 		for(c=0; c<ht->ncols; c++) {
 			db_cols[c] = &ht->scols[c];
 		}
-- 
2.11.0

